tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -196,13 +197,17 @@ def default_partition_fn(keys, shard_num):
             size of variable. If in training program, the variable is updated by
             optimizer, then the sparse slot variables in optimizer are also be
             restricted.
+          bp_v2: update parameters by *updating* instead of *setting*, which solves
```

Comment:
I think it could be better to tell the `bp_v2` is boolean in the annotation. Docs will be auto-generated from it.

Reply:
*I think it could be better to tell the bp_v2 is boolean in the annotation.*

Sorry, I'm not familiar with tfra doc generator, could you pls tell me what should I do exactly? 

What should I do to generate docs? and what is the *annotation*?

Reply:
thx u. comments updated.